src/cli.ts

```diff
@@ -34,11 +48,25 @@ import { Generator } from './generator';
     else
       options[name] = [];
   }
-  const rootDir = path.resolve(process.cwd(), args[0] || '');
   if (options.help) {
     _printHelp();
+    process.exit(0);
+  }
+
+  const bothBooleanOptionsSpecified = booleanOptionPairs.filter(
+    ([positive, negative]) => options[positive] && options[negative],
+  );
+  if (bothBooleanOptionsSpecified.length > 0) {
+    console.error(
+      `Cannot specify both of:
+  ${bothBooleanOptionsSpecified
+    .map(([positive, negative]) => `- ${positive} and ${negative}`)
+    .join('\n')}`,
+    );
     process.exit(1);
   }
```

Comment:
prevent both boolean options from being passed (eg. both `--no-browsers` and `--browsers`) - will work for all future boolean options